ci/travis/install-minimal.sh

```diff
@@ -15,3 +15,8 @@ eval "${WORKSPACE_DIR}/ci/travis/ci.sh build"
 python -m pip install -U \
   pytest==5.4.3 \
   numpy
+
+# Runtime Env Agent require async_timeout,
+# we don't need to add async_timeout on ray's requirement,
+# because aiohttp require async_timeout.
+python -m pip install async_timeout
```

Comment:
I think this is a problem actually.  I see that `aiohttp` is in `requirements.txt`, which is what developers use.  But in `setup.py`, `aiohttp` only appears as an "extra", in `ray[default]`.  That means `aiohttp` is not installed when users install minimal Ray (using `pip install ray`).  So if we merge this PR, runtime environments will no longer work with minimal Ray.

@richardliaw is it okay to add `async_timeout` to the minimal Ray dependencies?  If not, @Catch-Bull is there a way to work around it somehow and avoid using `async_timeout`?

Reply:
If not, I can copy the source from [async-timeout](https://github.com/aio-libs/async-timeout), they use Apache 2.0 License

Reply:
That sounds good.  If we go this route, could you please update the `LICENSE` file?